README.md

```
@@ -63,37 +167,51 @@ For Software:
 
-![Build](Add photos of build process here)
-*Explain the build steps*
+![Final]![home automation pic](https://github.com/user-attachments/assets/1ae7a35f-eb6e-4dfb-b475-0c04c6af9cfc)
+![home automation pic3](https://github.com/user-attachments/assets/fd299e07-ffee-4d37-b5c3-c28469205d9c)
+ it is a automated home model with smart led lighting ,gas sensor for safety,ultrasonic sensor for ehanced parking experience,servo motor for automatic door.
 
-![Final](Add photo of final product here)
-*Explain the final build*
 
-### Project Demo
+### Project Demor 
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)